### DISCUSSION TITLE
Is it possible to self-host Directus with Microsoft Azure?

### DISCUSSION THREAD

**Discussion by cristianmayo** [directus/directus#17751] [Archive]
I signed up for an account and understand the Community tier has been paused. I also received the communication email regarding this one, as I wanted to learn more about the platform. The blog post mentioned in the communication email mentioned the possibility of self-hosting it in my infrastructure. 

As I read more about Directus infrastructure, it is using AWS... However, is this possible with Microsoft Azure?

**Comment by haniefss** [comment]
You can definitely deploy Directus on Azure. I use Azure app service to host Directus (and blob storage and azure database for mysql for files and db).

**Reply by cristianmayo** [reply]
@haniefss Thanks! Would you mind me requesting for links (if there's any) from you? I would like to explore the platform with Azure services... :)
